rivers/jbossdeveloper_vimeo_drupal.json

```diff
@@ -0,0 +1,128 @@
+{
+  "type" : "remote",
+  "remote" : {
+    "urlGetDocuments"               : "http://{apiKey}",
+    "listDocumentsMode"             : "simple",
+    "spacesIndexed"                 : "MAIN",
+    "timeout"                       : "60s",
+    "indexUpdatePeriod"             : "10m",
```

Comment:
From yesterday it seems this setting doesn't work, please experiment with the full update one to see if it works as expected. Same for the second river config